src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -436,6 +439,10 @@ class boss_vazruden_the_herald : public CreatureScript
                                 return;
                             }
                         }
+                    if (!(Nazan && Nazan->IsAlive()) && !(Vazruden && Vazruden->IsAlive()))
```

Comment:
not sure if it needs more spaces xd

Reply:
Dang. I added 4 less spaces than there should have been. Fair enough, new fix incoming.